### ISSUE TITLE
Run on Startup does not work

### ISSUE THREAD

**Issue by iHMahmoodi** [seerge/g-helper#2824]
### Rules

- [X] I made myself familiar with the <a href='https://github.com/seerge/g-helper?tab=readme-ov-file'>Readme</a>, <a href='https://github.com/seerge/g-helper/wiki/FAQ'>FAQ</a> and <a href='https://github.com/seerge/g-helper/wiki/Troubleshooting'>Troubleshooting</a>.
- [X] I understand that, if insufficient information or no app logs will be provided, my issue will be closed without an answer.

### What's wrong?

Hi,
each time I turn on my computer, the app does not run automatically on startup.
each time, I checked this option's box, but nothing happened and did not add to the Windows startup list.

### How to reproduce the bug?

1- open app
2- check the box for run on startup
3- nothing happened and did not lunch on the next windows lunch

### Logs

[log.txt](https://github.com/user-attachments/files/16156304/log.txt)

### Device and Model

M16 2022

### Armoury Crate

Uninstalled

### OS

Windows 11 23H2

**Comment by seerge** [comment]
@iHMahmoodi hello,

That checkbox creates a new task in windows Task Scheduler pointing to exe.

1. Where do you keep exe ?
2. Post a screenshot of the task details in Task Scheduler 

Thanks

**Comment by iHMahmoodi** [comment]
![image](https://github.com/seerge/g-helper/assets/8407057/eec5d738-9566-459d-beee-0785576b1d34)
![image](https://github.com/seerge/g-helper/assets/8407057/b93de674-8e24-47ab-bcd0-67601815b687)
![image](https://github.com/seerge/g-helper/assets/8407057/bcd0e2e1-307a-4b6d-867e-bb5724bca3f4)


> @iHMahmoodi hello,
> 
> That checkbox creates a new task in windows Task Scheduler pointing to exe.
> 
> 1. Where do you keep exe ?
> 2. Post a screenshot of the task details in Task Scheduler
> 
> Thanks

**Comment by seerge** [comment]
@iHMahmoodi ok, great.

Do you have any strange permissions for that folder on H drive where you have placed exe ? (and is the path correct?)
Check if Task Scheduler shows any reasons why it didn't run it ?

Cause it looks like just windows can't run app by some own reasons ...

**Comment by iHMahmoodi** [comment]
> @iHMahmoodi ok, great.
> 
> Do you have any strange permissions for that folder on H drive where you have placed exe ? (and is the path correct?) Check if Task Scheduler shows any reasons why it didn't run it ?
> 
> Cause it looks like just windows can't run app by some own reasons ...

No, that's a normal foder without any permission

2 or 3 previous versions of GHelper just worked fine but after I replaced it with this new version, that does not work properly anymore.
i have a solution for that and I want to remove that task Scheduler , then check that box to create a new task.
if that works, I let you know

**Comment by seerge** [comment]
@iHMahmoodi you can just uncheck and check again `Run on startup` - that deletes and create a task again.

**Comment by iHMahmoodi** [comment]
> @iHMahmoodi you can just uncheck and check again `Run on startup` - that deletes and create a task again.

that check does not work at all
so, I remove the task from the task scheduler and then run the app to check the box for creating a new task.
but that's not work and it seems nothing happened after I removed the task

so I went to %appdata% and remove GHelper folder and the config file

I Open that app and check that box and now, that created a task successfully.
after I restart my system, the GHelper now works fine and started automatically.
